tests/cloner_test.go

```diff
@@ -508,6 +515,40 @@ var _ = Describe("all clone tests", func() {
 				err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
 				Expect(err).ToNot(HaveOccurred())
 			})
+
+			It("[rfe_id:1126][crit:High][vendor:cnv-qe@redhat.com][level:component] Should fail with Event when cloning into a smaller sized data volume", func() {
+				By("Creating a source from a real image") // todo fs....
+				sourceDv := utils.NewDataVolumeWithHTTPImport("source-dv", "200Mi", tinyCoreIsoURL())
+				filesystem := v1.PersistentVolumeFilesystem
+				sourceDv.Spec.PVC.VolumeMode = &filesystem
+				sourceDv, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, sourceDv)
+				fmt.Fprintf(GinkgoWriter, "sourceDv %v", sourceDv)
+
+				Expect(err).ToNot(HaveOccurred())
+				f.ForceBindPvcIfDvIsWaitForFirstConsumer(sourceDv)
+
+				By("Waiting for import to be completed")
+				utils.WaitForDataVolumePhaseWithTimeout(f.CdiClient, f.Namespace.Name, cdiv1.Succeeded, sourceDv.Name, 3*90*time.Second)
+
+				By("Cloning from the source DataVolume to under sized target")
+				targetDv := utils.NewDataVolumeForImageCloningAndStorageSpec("target-dv", "100Mi",
+					f.Namespace.Name,
+					sourceDv.Name,
+					sourceDv.Spec.PVC.StorageClassName,
+					sourceDv.Spec.PVC.VolumeMode)
+				fmt.Fprintf(GinkgoWriter, "targetDv %v", targetDv)
+
+				targetDv, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, targetDv)
+				Expect(err).ToNot(HaveOccurred())
+				// As of now, csi and smart clone check the values before the pvc is created, and the network clone
+				// checks it in the upload phase, so it needs a PVC, this might be improved
```

Comment:
so should a check a pvc was created or not should be added here?

Reply:
that is why the next line 2 lines are:  
```
				if cloneType == "network" {
					f.ForceBindPvcIfDvIsWaitForFirstConsumer(targetDv)
				}
```
Because only on network clone the pvc is created so we can try to consume it if WFFC. 
If you think the comment is not clear I should try to change the comment or the code.

Reply:
the function ForceBindPvcIfDvIsWaitForFirstConsumer wasnt clear to me that it also check that a pvc was created. that ok then

Reply:
yes, it works on a PVC so it really needs a PVC